style.css

```diff
@@ -0,0 +1,165 @@
+body {
+	width: 100%;
+	height: 100%;
+	margin: 0 auto;
+	background: url(images/Background.png);
+	background-size: cover;
+}
+
+.wrap {
+	font-family: Roboto, serif;
+	color: #fff;
+	padding: 20px;
+	max-width: 1280px;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 50%;
+  margin: 0 auto;
+  transform: translate(0, -50%);
+}
+
+.title {
+	font-size: 57px;
+}
+
+.subtitle {
+	font-size: 19px;
+	line-height: 35px;
+	margin-bottom: 60px;
+}
+
+.label {
+	position: relative;
+}
+
+.text {
+  display: block;
+	font-size: 50px;
+  padding-left: 41px;
+	margin-bottom: 24px;
+}
+
+.text:hover {
+	opacity: .8;
+	cursor: pointer;
+}
+
+.radio {
+  display: none;
+}
+
+.fake::before {
+  content: "";
+  position: absolute;
+  top: 12px;
+  width: 30px;
+  height: 30px;
+  background: url(images/Rectangle.png) left no-repeat;
+  opacity: 0;
+  transition: .2s;
+}
+
+.radio:checked + .fake::before {
+  opacity: 1;
+}
+
+.columns {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+
+.btn {
+	display: inline-block;
+	background-color: #d62b2e;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/evagame/pullRequest?prid=5788637)